server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -773,6 +767,27 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
 
     }
 
+    @Nullable
+    private AlertVO createAlertForTrivialAlertType(AlertType alertType, long dataCenterId, Long podId, Long clusterId) {
+        AlertVO alert = null;
+        List<AlertType> alerts = Arrays.asList(AlertType.ALERT_TYPE_HOST
```

Comment:
should we define this list in the class itself to allow it be created only once?